examples/backends/sglang/launch/agg_grpc.sh

```
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Aggregated serving via the Rust bridge to SGLang's native gRPC server.
+# Three processes: dynamo.frontend, sglang.launch_server, dynamo.sglang_grpc.
+# GPUs: 1
+
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)